plasmapy/physics/tests/test_parameters.py

```diff
@@ -477,39 +476,33 @@ def test_gyrofrequency():
 def test_gyroradius():
     r"""Test the gyroradius function in parameters.py."""
 
-    assert gyroradius(B, T_e).unit == u.m
+    assert gyroradius(B, T_i=T_e).unit == u.m
 
-    assert gyroradius(B, 25 * u.m / u.s).unit == u.m
-
-    assert gyroradius(T_e, B) == gyroradius(B, T_e)
-
-    assert gyroradius(V, B) == gyroradius(B, V)
-
-    assert gyroradius(B, V) == gyroradius(B, -V)
+    assert gyroradius(B, Vperp=25 * u.m / u.s).unit == u.m
 
     Vperp = 1e6 * u.m / u.s
     Bmag = 1 * u.T
     omega_ce = gyrofrequency(Bmag)
-    assert gyroradius(Bmag, Vperp) == \
+    assert gyroradius(Bmag, Vperp=Vperp) == \
```

Comment:
This is trying to do too much in one line. Split it up and get rid of the continuation slash.